PRS and ERS don't promote replicas taking backups (#16997)
vitessio/vitess: changelog/22.0/22.0.0/summary.md

## Summary

### Table of Contents

- **[Known Issues](#known-issues)**
- **[Major Changes](#major-changes)**

>     - **[Support for specifying expected primary in reparents](#reparents-prefer-not-backing-up)**

Reviewer: ```suggestion
    - **[Avoid promoting a replica that is currently taking a backup](#reparents-prefer-not-backing-up)**
```

## <a id="known-issues"/>Known Issues</a>

## <a id="major-changes"/>Major Changes</a>

> ### <a id="reparents-prefer-not-backing-up"/>Support specifying expected primary in reparents

Reviewer: ```suggestion
### <a id="reparents-prefer-not-backing-up"/>Avoid promoting a replica that is currently taking a backup
```

> The execution of both Planned Shard Reparents and Emergency Reparents now prefer promoting replicas that are not

Reviewer: ```suggestion
Both planned and emergency failovers now prefer promoting replicas that are not
```

> currently taking backups. Notice that if there's only one suitable replica to promote, and it is taking a backup, it

Reviewer: ```suggestion
currently taking backups. Note that if there's only one suitable replica to promote, and it is taking a backup, it
```

> will still be promoted.

Reviewer: ```suggestion
will still be promoted. This does not apply to `builtin` backups. A replica that is currently taking a `builtin` backup will never be promoted.
```
